### ISSUE TITLE
Unable to configure Spectral rule for `oas3-valid-example`

### ISSUE THREAD

**Issue by rparini** [IBM/openapi-validator#364]
Using v0.53.1 with a `.spectral.yaml`

```yaml
extends: "@ibm-cloud/openapi-ruleset"
rules:
  oas3-valid-example: error
```
results in:
```
[Info] Problem reading Spectral config file, using default IBM Spectral ruleset. Cause for error:
Cannot extend non-existing rule: "oas3-valid-example"
```

I am new to this validation library and Spectral so may be misunderstanding something but this rule does currently exist in [Spectral's documentation](https://github.com/stoplightio/spectral/blob/develop/docs/reference/openapi-rules.md#oas3-valid-example) and in [IBM openapi-validator documentation](https://github.com/IBM/openapi-validator/blob/a71c72b186acff84a98283ae1165309dec51d8ef/README.md?plain=1#L523) so it seems like a user should be able to set it?

**Comment by dpopp07** [comment]
Hi @rparini - thanks for the issue. It looks like both our documentation and Spectral's documentation is out of date. Although they were quite unclear about it, Spectral separated this rule into the rules `oas3-valid-media-example` and `oas3-valid-schema-example` in their last major release.

This is hinted at in [their migration guide](https://meta.stoplight.io/docs/spectral/ZG9jOjg2MDIwMDM-spectral-v5-to-v6-migration-guide#oas) although I see there is no mention of `oas3-valid-example` as part of the change. I confirmed these are the correct new rules in the [source code for their ruleset](https://github.com/stoplightio/spectral/blob/develop/packages/rulesets/src/oas/index.ts).

Hopefully using these new rules resolves your issue. In the meantime, I will update our documentation.

**Comment by rparini** [comment]
Hi @dpopp07 thanks very much for the quick response and the links.  I'll make a similar issue on Spectral's repo to prompt them to update their documentation as well. 

Happy for this issue to be closed, if you prefer, or for it to be left open until the IBM validator's documentation is updated.

**Comment by dpopp07** [comment]
Sounds good! I'll leave this open until I merge in the documentation changes. Thanks again
